<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryIndexDialog</class>
 <widget class="QDialog" name="EntryIndexDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit index entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="entryLabel">
     <property name="text">
      <string>Entry:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ReplaceLineEdit" name="entryEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="descriptionLabel">
     <property name="text">
      <string>Description:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="ReplaceTextEdit" name="commentEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="commentLabel">
     <property name="text">
      <string>Comment:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="urlEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>10</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="urlLabel">
     <property name="text">
      <string>URL:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="ReplaceTextEdit" name="descriptionEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReplaceLineEdit</class>
   <extends>QLineEdit</extends>
   <header>replacelineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ReplaceTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>replacetextedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>entryEdit</tabstop>
  <tabstop>urlEdit</tabstop>
  <tabstop>commentEdit</tabstop>
  <tabstop>descriptionEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntryIndexDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntryIndexDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
